### DISCUSSION TITLE
install coder in docker,got coder agent download error.

### DISCUSSION THREAD

**Discussion by manooog** [coder/coder#16301] [General]
use docker template to create workspace:
![image](https://github.com/user-attachments/assets/e88f4c3d-192c-4600-92de-efdad3c94249)

as you can see,the workspace is start up as a container:
![image](https://github.com/user-attachments/assets/6309b3ea-0a66-4c50-8ddb-09fc14c6b785)

but the workspace can not download agent?
```
+ curl -fsSL --compressed https://9jcsovs2h0rkg.pit-1.try.coder.app/bin/coder-linux-amd64 -o coder
curl: (22) The requested URL returned error: 502
error: failed to download coder agent
       command returned: 22
Trying again in 30 seconds...
```

**Comment by stirby** [comment]
Hey @manooog, are you using the docker example template? I assume you're on version 2.14.2, what OS are you using?

**Comment by tianzhanshen** [comment]
﻿﻿﻿Yes, I also have this problem. I keep getting stuck with 502 and can't remove 
![Uploading coder.jpg…]()

**Comment by tianzhanshen** [comment]
![coder](https://github.com/user-attachments/assets/bbd709ba-a6ff-4012-b9c8-a0202aaa5078)

**Comment by ron-zhong** [comment]
Having the same issue here:
- control plane: Ubuntu 20.04 LTS (in Windows)
- workplace template: Docker Container

![Image](https://github.com/user-attachments/assets/a31a1c20-095b-4f7b-8964-f3f1e23a4fd5)

![Image](https://github.com/user-attachments/assets/69f73bc4-1964-4ed5-86a7-56770c7a6c1a)

![Image](https://github.com/user-attachments/assets/72dcface-5857-4472-bac6-293c9b6dd2f9)

![Image](https://github.com/user-attachments/assets/4cd4eb54-99fe-4133-afa8-45a47ad4ead9)
